### ISSUE TITLE
Weight Normalization

### ISSUE THREAD

**Issue by evanatyourservice** [philipperemy/keras-tcn#91]
tensorflow has an easy to use weight normalization wrapper here:

https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/wrappers.py

**Comment by philipperemy** [comment]
@evanatyourservice interesting. Might want to prioritize that once we run on tensorflow.keras instead of keras.

**Comment by philipperemy** [comment]
https://github.com/philipperemy/keras-tcn/pull/96

**Comment by philipperemy** [comment]
Closed with https://github.com/philipperemy/keras-tcn/commit/1efc54cb5d8c6997d2436d5222fbf9685f2c11af.

I'll release on PyPI soon :)
